src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useMemo, useState } from 'react';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { ErrorLoading } from '../components/ErrorLoading';
+import { NoPeople } from '../components/NoPeople';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState<boolean>(false);
```

Comment:
you could initially set loading to true, and when you're done fetching your data - set it to false